client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -354,7 +354,6 @@ const siteSetupFlow: Flow = {
 						case SiteIntent.DIFM:
 							return navigate( 'difmStartingPoint' );
 
-						case SiteIntent.Write:
```

Comment:
Although I prefer a simplified approach, unfortunately this didn't work on my tests.
Any special steps to reproduce the fix